src/Sylius/Component/Order/Model/Adjustment.php

```diff
@@ -104,8 +89,6 @@ public function getAdjustable()
         if (null !== $this->orderItem) {
             return $this->orderItem;
         }
-
-        return null;
```

Comment:
In Sylius we are always explicit, so the `return null` should always end a method which returns something. I think we had this discussion in the past.